docs/guides/not-found.md

```diff
@@ -11,7 +11,7 @@ There are two primary cases where a Remix site should send a 404:
 - The URL doesn't match any routes in the app
 - Your loader didn't find any data
 
-The first case is already handled by Remix, you don't have to do anything. It knows your routes so it knows if nothing matched. The second case is up to you, but it's really easy.
+The first case is already handled by Remix, you don't have to throw an error yourself. It knows your routes so it knows if nothing matched (_consider using a [Splat Route][splat-route] to handle this case_). The second case is up to you, but it's really easy.
```

Comment:
"you don't have to **throw a response**" is correct here. Error and response are totally different thing in this context.

Reply:
yes good point, I'll clarify it to your suggestion